yrstats.py

```diff
@@ -160,7 +145,11 @@ def do(self, event):
 
         self.last_notification_time = time_now
 
-        gamestats = call_stat_dmp_parser(self.config, stat_dmp_file_path = event.src_path)
+        gamestats = statparser.process_stats(
```

Comment:
can you give an option to call the php based stat parser vs the python stat parser or are you confident that yours supports all the features/details extracted by the old one?

Reply:
done